app/session_flows/coronavirus_find_support_flow.rb

```diff
@@ -1,32 +1,85 @@
 class CoronavirusFindSupportFlow
-  NODES = {
-    need_help_with: :feel_safe,
-    feel_safe: :afford_rent_mortgage_bills,
-    afford_rent_mortgage_bills: :afford_food,
-    afford_food: :get_food,
-    get_food: :able_to_go_out,
-    able_to_go_out: :self_employed,
-    self_employed: :have_you_been_made_unemployed,
-    have_you_been_made_unemployed: :are_you_off_work_ill,
-    are_you_off_work_ill: :worried_about_work,
-    worried_about_work: :have_somewhere_to_live,
-    have_somewhere_to_live: :have_you_been_evicted,
-    have_you_been_evicted: :mental_health_worries,
-    mental_health_worries: :nation,
-    nation: :results,
-    results: :unknown,
-  }.freeze
-
-  attr_reader :node_name
-  def initialize(node_name)
+  class << self
+    def option_questions
+      {
+        feeling_unsafe: :feel_safe,
+        paying_bills: :afford_rent_mortgage_bills,
+        getting_food: :afford_food,
+        being_unemployed: :self_employed,
+        going_to_work: :worried_about_work,
+        somewhere_to_live: :have_somewhere_to_live,
+        mental_health: :mental_health_worries,
+        not_sure: :nation,
+      }
+    end
+  end
+
+  delegate :option_questions, to: :class
+
+  attr_reader :node_name, :session
+  def initialize(node_name, session = {})
     @node_name = node_name.to_sym
+    @session = session || {}
   end
 
   def next_node
-    NODES[node_name]
+    nodes[node_name]
+  end
+
+  def nodes
+    {
+      need_help_with: next_group_start,
+      feel_safe: next_group_start,
+      afford_rent_mortgage_bills: next_group_start,
+      afford_food: :get_food,
+      get_food: node_after_get_food,
+      able_to_go_out: :self_employed,
+      self_employed: node_after_self_employed,
+      have_you_been_made_unemployed: node_after_have_you_been_made_unemployed,
+      are_you_off_work_ill: next_group_start,
+      worried_about_work: next_group_start,
+      have_somewhere_to_live: :have_you_been_evicted,
+      have_you_been_evicted: next_group_start,
+      mental_health_worries: :nation,
+      nation: :results,
+    }
   end
```

Comment:
Hi @reggieb Nice!  I like this approach, much cleaner.  And all the flow logic is in one place.